Source/Core/EarthOrientationParameters.js

```diff
@@ -100,7 +100,7 @@ import TimeStandard from './TimeStandard.js';
      */
     EarthOrientationParameters.NONE = Object.freeze({
             getPromiseToLoad : function() {
-                return when();
+                return when.resolve();
```

Comment:
The old call would never resolve, which didn't make sense since NONE means loading is done.